apps/web/src/ee/billing/components/Features.tsx

```diff
@@ -60,7 +60,6 @@ type Feature = {
 const features: Feature[] = [
   {
     label: 'Platform',
-    isContrast: false,
```

Comment:
This was refactored so that we never have to adjacent background colors that are the same.